cadence/contracts/bridge/FlowEVMBridgeHandlers.cdc

```diff
@@ -191,14 +192,229 @@ access(all) contract FlowEVMBridgeHandlers {
+            assert(
+                postBalance > preBalance,
+                message: "Escrowed FLOW Balance did not increment after unwrapping WFLOW - pre: ".concat(preBalance.toString())
+                    .concat(" | post: ").concat(postBalance.toString())
+            )
+            // Confirm bridge COA's FLOW balance has incremented by the expected amount
+            assert(
+                UInt256(postBalance - preBalance) == amount,
+                message: "Escrowed WFLOW balance after unwrapping does not match requested amount - expected: "
+                    .concat((UInt256(preBalance) + amount).toString())
+                    .concat(" | actual: ")
+                    .concat((postBalance - preBalance).toString())
+            )
+
+            // Withdraw escrowed FLOW from bridge COA
+            let withdrawBalance = EVM.Balance(attoflow: UInt(amount))
+            assert(
+                UInt256(withdrawBalance.attoflow) == amount,
+                message: "Requested balance failed to convert to attoflow - expected: "
+                    .concat(amount.toString())
+                    .concat(" | actual: ")
+                    .concat(withdrawBalance.attoflow.toString())
+            )
+            let flowVault <- coa.withdraw(balance: withdrawBalance)
+            assert(
+                flowVault.balance == ufixAmount,
+                message: "Resulting FLOW Vault balance does not match requested amount - expected: "
+                    .concat(ufixAmount.toString())
+                    .concat(" | actual: ")
+                    .concat(flowVault.balance.toString())
+            )
+            return <-flowVault
+        }
+
+        /* --- HandlerAdmin --- */
+        // Conforms to HandlerAdmin for enableBridging, but most of the methods are unnecessary given the strict
+        // association between FLOW and WFLOW
+
+        /// Sets the target type for the handler
+        access(FlowEVMBridgeHandlerInterfaces.Admin)
+        fun setTargetType(_ type: Type) {
+            panic("WFLOWTokenHandler has targetType set to "
+                .concat(self.targetType.identifier).concat(" at initialization"))
+        }
+
+        /// Sets the target EVM address for the handler
+        access(FlowEVMBridgeHandlerInterfaces.Admin)
+        fun setTargetEVMAddress(_ address: EVM.EVMAddress) {
+            panic("WFLOWTokenHandler has EVMAddress set to "
+                .concat(self.targetEVMAddress.toString()).concat(" at initialization"))
+        }
+
+        /// Sets the target type for the handler
+        access(FlowEVMBridgeHandlerInterfaces.Admin)
+        fun setMinter(_ minter: @{FlowEVMBridgeHandlerInterfaces.TokenMinter}) {
+            panic("WFLOWTokenHandler does not utilize a minter")
+        }
+
+        /// Enables the handler for request handling. The
+        access(FlowEVMBridgeHandlerInterfaces.Admin)
+        fun enableBridging() {
+            self.enabled = true
```

Comment:
do we need a disable function too?

Reply:
I suppose we could add that here, though FlowEVMBridgeConfig allows pausing on a Type basis as well

Reply:
Added in [769a6a1](https://github.com/onflow/flow-evm-bridge/pull/137/commits/769a6a111c2787e5940596d4f3b07042d42d4968)